packages/loader/container-loader/src/container.ts

```diff
@@ -132,6 +132,10 @@ export type DetachedContainerSource = {
     create: false,
 };
 
+interface IContainerClient extends IClient{
+    readonly containerId: string;
```

Comment:
Some notes:
1. Given that it's not exported, and Continer.client() is not public, not clear who is it for an what is the usage patter,
2. Comment above it and field would be nice.
3. I'd suggest to rename field to containerInstanceId, to slightly better reflect semantics, as "container id" can be interpreted in so many ways... ("instance" still has similar issue, but it's slightly better IMHO).